src/components/SecondaryPanes/index.js

```diff
@@ -215,12 +215,24 @@ class SecondaryPanes extends Component {
     });
   }
 
+  renderVerticalCommandbar() {
+    return dom.div(
+      {
+        className: "",
+        style: {
+          width: "100vw"
```

Comment:
lets move this to CSS.

Perhaps we should pass layout to `CommandBar` so that it can have a vertical, horizontal class?